### PR TITLE
Add order lines to API Spec

### PR DESCRIPTION
**Pull request by etmartinkazoo**
This adds customer and internal notes to the API Spec on Order Lines.

### CODE REVIEW

File: spec.json [hunk 1]
```diff
@@ -11697,6 +11697,14 @@
...
               "type": "string"
             }
+          },
+          "customer-note": {
+            "description": "A string of text concerning the customer to whom the order-line is associated.",
+            "type": "string"
+          },
+          "internal-note": {
+            "description": "A string of text concerning the customer to whom the order-line is associated entered internally by a superuser.",
+            "type": "string"
```
Comment (davidmcatee-azure) [inline spec.json:11707]: Can you make these camelCase please?

File: spec.json [hunk 2]
```diff
@@ -11705,7 +11713,9 @@
           "name",
           "quantity-ordered",
           "weight",
-          "volume"
+          "volume",
+          "customer-note",
+          "internal-note"
```
Comment (davidmcatee-azure) [inline spec.json:11718]: I don't think these should be required. If there aren't any notes, they would be omitted.

**Review by davidmcatee-azure — CHANGES_REQUESTED** [review]
A general question: Is the UI for entering internal notes going to be the website in superuser mode?

File: spec.json (hunk 1 above)
Comment (etmartinkazoo) [reply spec.json]: @davidmcatee-azure - Yes, I've changed to camelCase and committed.

File: spec.json (hunk 2 above)
Comment (etmartinkazoo) [reply spec.json]: Thank you, yes you are right. Removed and committed.

**Comment by etmartinkazoo** [comment]
> A general question: Is the UI for entering internal notes going to be the website in superuser mode?

Yes, superusers will be able to enter internal notes on the website.

**Review by davidmcatee-azure — APPROVED** [review]

**Comment by etmartinkazoo** [comment]
@davidmcatee-azure - For the API spec is it sufficient to merely `Merge` the PR into master or is there more to do vis a vis deployment?

**Review by tommckennon — APPROVED** [review]

**Comment by davidmcatee-azure** [comment]
> @davidmcatee-azure - For the API spec is it sufficient to merely `Merge` the PR into master or is there more to do vis a vis deployment?

Just merging to master here. I can do so.

**Pull request merged**